### PR TITLE
Added route for Simple Update action.

### PR DESCRIPTION
**Pull request by ehaligow**
Added route for Simple Update action.

### CODE REVIEW

File: svc-device-manager/rest/postSimpleUpdate.go [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+		logrus.Error(missingInfoError)
+		ctx.StatusCode(http.StatusBadRequest)
+		ctx.WriteString(missingInfoError)
+		return
+	}
+
+	// Validate Simple Update request
+	err = validateSimpleUpdateAction(reqInfo.Body)
+	if err != nil {
+		simpleUpdateRequestError := "invalid simple update request: %s" + err.Error()
```
Comment (MaciejMis) [inline svc-device-manager/rest/postSimpleUpdate.go:43]: Using a verb `%s` without a `Sprintf` or any other formatting method

File: svc-device-manager/rest/postSimpleUpdate.go [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+	if err != nil {
+		errorMessage := "Error while reading response body: " + err.Error()
+		logrus.Error(errorMessage)
+		ctx.StatusCode(http.StatusInternalServerError)
+		ctx.WriteString(errorMessage)
+		return
+	}
+
+	ctx.StatusCode(response.StatusCode)
+	ctx.Write(body)
```
Comment (MaciejMis) [inline svc-device-manager/rest/postSimpleUpdate.go:75]: According to the "Redfish Firmware Update White Paper" 6.1 paragraph, service can respond with `202 Accepted` which means that a task is returned in `Location` header. Please forward this task URI in header.

**Review by MaciejMis — CHANGES_REQUESTED** [review]

File: svc-device-manager/rest/postSimpleUpdate.go (hunk 1 above)
Comment (ehaligow) [reply svc-device-manager/rest/postSimpleUpdate.go]: Done.

**Review by MaciejMis — APPROVED** [review]

**Comment by MaciejMis** [comment]
Accepted on 13.06.2023 in community meeting.

**Pull request merged**